doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -893,6 +912,7 @@ Libraries for networking
   * Updated the device mode callback to send an event when the provisioning state changes.
   * Fixed file descriptor handling by setting the :c:struct:`coap_client` structure's ``fd`` field to ``-1`` when closing the socket.
   * Added the :kconfig:option:`CONFIG_NRF_PROVISIONING_PRINT_ATTESTATION_TOKEN` option to enable printing the attestation token when the device is not yet claimed.
+  * Regenerate cbor code with zcbor 0.8.1 and adapt to API changes in zcbor 0.8.x.
```

Comment:
```suggestion
  * Added the :kconfig:option:`CONFIG_NRF_PROVISIONING_PRINT_ATTESTATION_TOKEN` option to enable printing the attestation token when the device is not yet claimed.
  * Updated:
  
    * The device mode callback sends an event when the provisioning state changes.
    * Regenerated cbor code with zcbor 0.8.1 and adapted to API changes in zcbor 0.8.x.

  * Fixed file descriptor handling by setting the :c:struct:`coap_client` structure's ``fd`` field to ``-1`` when closing the socket.
```

Reply:
Your last proposal changes text unrelated to my changes, can you confirm that I should change it here?

Reply:
It is just grouping related points together. I can fix this later if needed.

Reply:
No worries, I added it, just wanted to confirm.